RetailCoder.VBE/Settings/CodeInspectionConfigProvider.cs

```diff
@@ -76,7 +76,7 @@ public HashSet<CodeInspectionSetting> GetDefaultCodeInspections()
                 new CodeInspectionSetting("RedundantOptionInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Hint),
                 new CodeInspectionSetting("SelfAssignedDeclarationInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Suggestion),
                 new CodeInspectionSetting("UnassignedVariableUsageInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Error),
-                new CodeInspectionSetting("UndeclaredVariableUsageInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Error),
+                new CodeInspectionSetting("UndeclaredVariableInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Error),
```

Comment:
this doesn't look right .... why is this even here still? @retailcoder please check PRs before merging, we did remove this mess already..